scripts/descriptors/flotsamgenerator.lua

```diff
@@ -0,0 +1,30 @@
+local spawn_task
+if TheWorld.components.flotsamgenerator and TheWorld.components.flotsamgenerator.ScheduleGuaranteedSpawn then
+	spawn_task = util.getupvalue(TheWorld.components.flotsamgenerator.ScheduleGuaranteedSpawn, "_guaranteed_spawn_tasks")
+end
+
+local function Describe(self, context)
+	local descriptions = {}
+	if spawn_task then
+		for p, tasks in pairs(spawn_task) do
+			for v, task in pairs(tasks) do
+				if v.prefabs[1] == "messagebottle" then
+					table.insert(descriptions, {
```

Comment:
Descriptions coming from TheWorld.components need to have ``worldly = true,`` as part of the returned description.

In this specific instance, I will say that you should also have ``playerly = true`` for the upcoming support I'll be adding, mentioned in the PR.